src/feature_buffer.rs

```diff
@@ -334,6 +335,21 @@ impl FeatureBufferTranslator {
                     }
                 }
             }
+
+	    // rehash part
```

Comment:
Let's change this comment to explain that this is an implementation of fibhash and a add link to documentation